cinema_api/settings.py

```diff
@@ -0,0 +1,131 @@
+"""
+Django settings for cinema_service project.
+
+Generated by 'django-admin startproject' using Django 4.0.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = (
+    "django-insecure-ru6ndfka@95_(lysua8yhdjq@vpiqgv3yru4r)q3h4_u8x7dfy"
+)
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
Ensure that the `SECRET_KEY` is kept secret and not exposed in version control. Also, remember to set `DEBUG = False` in a production environment to avoid exposing sensitive information.